lib/rubocop/config_validator.rb

```diff
@@ -102,11 +102,21 @@ def check_target_ruby
     end
 
     def alert_about_unrecognized_cops(invalid_cop_names)
+      unknown_cops = list_unknown_cops(invalid_cop_names)
+
+      if ConfigLoader.ignore_unrecognized_cops
+        warn Rainbow('Ignoring unrecognized cops or departments by option.').yellow
+        warn unknown_cops.join("\n")
+      elsif unknown_cops.any?
+        raise ValidationError, unknown_cops.join("\n")
+      end
+    end
+
+    def list_unknown_cops(invalid_cop_names)
       unknown_cops = []
       invalid_cop_names.each do |name|
         # There could be a custom cop with this name. If so, don't warn
         next if Cop::Registry.global.contains_cop_matching?([name])
-
```

Comment:
This looks like an intentional blank line.

Reply:
Oh, sorry that is a mistake.